index.html

```diff
@@ -14,38 +14,38 @@ <h1 class="text-center" id="workWithUs">Work with us</h1>
     <div class="padded-content workwithusContent">
       <div class="row workwithusIconsRow col-md-10">
         <div class="col-md-3">
-          <img class="workwithusIcon" src="/images/Developers.svg" alt="Code brackets">
+          <img class="workwithusIcon" src="/images/developers.svg" alt="Code brackets">
```

Comment:
These updated icons are all black. Should they be red instead (per the design)?

Reply:
I tried changing the color to $cfa-red, but they are still black. Any suggestions?

Reply:
Sure! How did you try changing color? SVGs have special CSS properties. Have you tried targeting the SVG itself, and changing their `fill` and/or `stroke` properties?

Reply:
I was able to change the color of the icons.

Reply:
Woohoo! I noticed that the calendar icon still used black for the date boxes. Can that be updated as well?

<img width="214" alt="Screen Shot 2021-02-18 at 8 54 51 AM" src="https://user-images.githubusercontent.com/9326234/108374929-1a621c00-71c7-11eb-985f-25cc5bd266a7.png">

Reply:
Just wanted to pass along a tip. I looked at the SVG file for this icon and I would suggest adding a `fill="#CF2045"` attribute to the `<g>` (group) tag that wraps all of the SVG elements in this icon. Any attribute added to the group tag gets applied to all of its children.

Reply:
OH, thanks for the tip! It works. The whole icon is red now.